udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -101,6 +102,14 @@ <h1 class="fr-mb-0 fr-h1">
             {% endif %}
         </div>
     </div>
+    {% if external_source %}
+        {{ banner_info(
+            "fr-fi-external-link-line",
+            _("This dataset come from an external portal.
```

Comment:
I think we're missing a space between the two phrases here
![image](https://user-images.githubusercontent.com/28541902/153881396-da437c4f-8307-4525-997b-44d245709cea.png)